src/openzeppelin/account/Account.cairo

```diff
@@ -95,7 +95,8 @@ func __execute__{
         syscall_ptr : felt*,
         pedersen_ptr : HashBuiltin*,
         range_check_ptr,
-        ecdsa_ptr: SignatureBuiltin*
+        ecdsa_ptr: SignatureBuiltin*,
+        bitwise_ptr: BitwiseBuiltin*
```

Comment:
It looks like this is required for `unsafe_execute` since `eth_execute` also invokes it (with the implicit `bitwise_ptr`). The compiler will yell about mismatched implicit args if the pointer is removed here and in Account.execute. I don't think there's a performance impact because the `bitwise_builtin` will be 0 for vanilla Account executions. Not sure about steps though. It might be worth looking into